### ISSUE TITLE
Questions about training from scratch and configs

### ISSUE THREAD

**Issue by rlct1** [NVlabs/Sana#330]
Thank you so much for the great work! I am experimenting Sana with lightweight autoencoder and text encoder. Regarding training Sana from scratch, could you please share some insights and/or more information on the training configs? For example, in the technical report of Sana 1.0 (https://arxiv.org/pdf/2410.10629), Table 12: **Performance of Sana block design space**, you mentioned `all the models with the same training setting with 52K iterations`. Does it mean the 600M Sana can be trained with 52K iterations based on this config file (https://github.com/NVlabs/Sana/blob/main/configs/sana_config/512ms/Sana_600M_img512.yaml)? If yes and you used 32 gpus, 52K * batch size 128 * 32 gpus = ~213M images (and in SANA 1.5 report, you mentioned the pre-training dataset size is ~50M, right?) are sufficient to train Sana 600M on 512px and offers FID and CLIP = 6.4 and 27.4 respectively? 

Apart from the training iterations, batch size and the number of gpus, I am also curious about the learning rate and flow shift. Are you using constant learning rates (1e-4 for pretraining and 2e-5 for fine-tuning) and fixed flow shift 3.0 throughout all the trainings? I tried to increase the pretraining learning rate and the model learned faster. I wonder you tried other lr schedulers like cosine and cosine decay before. Similarly for the flow shift, it could be useful to improve image quality by sliding the flow shift back to 1.0 at later stage of pretraining/fine-tuning.

Finally, one more question about the chi prompt. Are you using the same chi prompt for both training and inferencing? If so, how do you handle misaligned enhanced prompts and training images? For example, if you have a cat image with a simple prompt "a cat sleeping", by feeding this simple prompt to the text encoder (e.g., gemma-2-2b-it) with the chi prompt, the enhanced prompt could be `A small, fluffy white cat curled up in a round shape, sleeping peacefully on a warm sunny windowsill, surrounded by pots of blooming red flowers.` but the generated information may not be presented in the corresponding training cat image. Would it complicate the training as there could be non-exist information embedded in the contextual text embedding condition?

I would be very appreciated if you can provide any useful information about training Sana from scratch. Thanks in advance!
